app/views/layouts/application.html.erb

```diff
@@ -1,27 +1,23 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>HomebrewBuddy</title>
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
+<head>
+  <%= csrf_meta_tag %>
```

Comment:
generated right? I'm curious about the switch to `csrf_meta_tag` (singular) from `csrf_meta_tags` (plural).

I have that same feeling for most of these generated changes.

Reply:
Hmm. [https://api.rubyonrails.org/classes/ActionView/Helpers/CsrfHelper.html#method-i-csrf_meta_tags](url) It looks like they are aliases for one another